### DISCUSSION TITLE
Unable to complete synchronization

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#5925] [General] [answered]
I am trying to upload a dgn to iModelhub using iTwin Synchronizer.  iTwin Synchronizer reported that synchronization completed successfully.  When I pressed 'Go To iModel' button, the 'View' button is disabled (grayed out).  

![Screen-1](https://github.com/iTwin/itwinjs-core/assets/83007711/1a5daa8f-1283-49a8-8dd9-b160c7522c88)

I logged in to iModelManager on Connect and when try to view the model, I get the following screen

![Screen-2](https://github.com/iTwin/itwinjs-core/assets/83007711/629214cd-b024-46e3-82e3-f7c26f0ad259)

I am able to see the model in iModelConsole but not able to view it (after 2 hrs, it still says it is 'Preparing...').  This is a small file with around 15-20 elements.

![image](https://github.com/iTwin/itwinjs-core/assets/83007711/6f910edb-1cc1-43b4-9bc2-81ca48d311a5)

**Comment by ovaisa** [comment]
I am now getting this message (since 2 days) in iTwinSynchronizer

![image](https://github.com/iTwin/itwinjs-core/assets/83007711/446fe175-d33e-4794-ab4d-98d58285ffc1)

iTwinSynchronizer shows the Sign In screen.  After I press the Sign In button, it briefly displays 'Please sign in the browser and then return here' .  After this 'Loading projects...' message is displayed briefly and then the message shown above comes up.  Control is not redirected to browser at any time.

I am unable to do anything after this message is displayed.

**Comment by harshalsc** [comment]
Hi @ovaisa Please share the logs so we can confirm the cause. Please click on gear icon on the top right **Gear icon -> Save logs** and send us the log file.
In the meanwhile, can you try logging out of the application and sign in again?

**Reply by ovaisa** [reply, marked as answer]
Its working correctly now and I am unable to upload the model.

Thanks,

Ovais

**Reply by harshalsc** [reply]
> Its working correctly now and I am unable to upload the model.

Hi @ovaisa , it is working but you are still unable to upload the model?

**Reply by ovaisa** [reply]
Apologies, my mistake a typo.  I am able to load the model and all is working as expected end to end.  Sorry for confusion!
